python_modules/dagster/dagster/core/execution/plan/active.py

```diff
@@ -119,24 +119,24 @@ def __exit__(self, exc_type, exc_value, traceback):
             )
             if self._interrupted:
                 raise DagsterExecutionInterruptedError(
-                    f"Execution of pipeline was interrupted before completing the execution plan. {state_str}"
+                    f"Execution of job/pipeline was interrupted before completing the execution plan. {state_str}"
```

Comment:
even with `JobDefinition`, will we not know if it's a job or a pipeline?

Reply:
Ah we actually can get the target def off of `ActiveExecution`, thanks for the catch